reddit_gold/controllers.py

```diff
@@ -46,7 +46,7 @@ def GET_partners(self):
         vuser=VExistingUname("username"),
     )
     def GET_snoovatar(self, vuser):
```

Comment:
There are a couple more checks that affect snoovatars due to the `VGold` validator on some of the endpoints below

[VGold](https://github.com/reddit/reddit-public/blob/master/r2/r2/lib/validator/validator.py#L1391-L1397)

Reply:
can't save my snoovatar cuz this https://github.com/reddit/reddit-plugin-gold/blob/master/reddit_gold/controllers.py#L82